ext/cache/01_cache.js

```diff
@@ -134,6 +134,22 @@ class Cache {
       throw new TypeError("Response body is already used");
     }
 
+    // If response body is a string or Uint8Array, attempt to auto-detect content-length
+    const headerList = [...innerResponse.headerList];
```

Comment:
This seems expensive to always create a copy of `.headerList`, even if the following if-statement doesn't come into effect.

Reply:
Fixed